spring-cloud-kubernetes-client-autoconfig/src/main/java/org/springframework/cloud/kubernetes/client/KubernetesClientPodUtils.java

```diff
@@ -64,11 +64,6 @@ public KubernetesClientPodUtils(CoreV1Api client, String namespace) {
 		return this.current;
 	}
 
-	@Override
```

Comment:
in general, this class looks very much like `Fabric8PodUtils`. if you agree on the changes in the latter, I will backport the same changes in the former.